README.md

```diff
@@ -42,6 +42,7 @@ Returns a Promise of the image URI.
     - `"base64"`: encode as base64 and returns the raw string. Use only with small images as this may result of lags (the string is sent over the bridge). *N.B. This is not a data uri, use `data-uri` instead*.
     - `"data-uri"`: same as `base64` but also includes the [Data URI scheme](https://en.wikipedia.org/wiki/Data_URI_scheme) header.
  - **`filename`** *(string)*: the name of the generated file if any (Android only). Defaults to `ReactNative_snapshot_image_${timestamp}`.
+ - **`isScrollView`** *(bool)*: View is evaluated the real height when the value is true.
```

Comment:
I'm not sure about setting that on JS side, it seems redundant information with checking `view instanceof ScrollView` ? OR at least, the naming is not good ('isScrollView' can be determined without using to tell).

I guess such a argument is still a good idea to have on JS side, because you want to choose between one of these option: snapshot the ScrollView actual display (default) vs the underlying 'container' size?

then maybe you need to rename to something like `snapshotContentContainer` . to reflect the 'contentContainer' naming convention of RN ScrollView. maybe a better name?